### PR TITLE
[Model Monitoring] Test and Fix for Model-monitoring data-path: support LLM/deep-learning models

### PR DESCRIPTION
**Pull request by davesh0812**
Implements and test [ML-5459](https://iguazio.atlassian.net/browse/ML-5459)

One important change : no more saving prediction as a vec or as a dict to the parquet (no one used it), only each port to separated column according to the model schema (if there is no schema the stream pod generated it).

[ML-5459]: https://iguazio.atlassian.net/browse/ML-5459?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tests/system/model_monitoring/assets/models.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 Iguazio
```
Comment (gtopper) [inline tests/system/model_monitoring/assets/models.py]:
```suggestion
# Copyright 2024 Iguazio
```

File: tests/system/model_monitoring/test_app.py [hunk 2]
```diff
@@ -450,3 +454,169 @@ def test_enable_model_monitoring(self) -> None:
...
             == "1m"
         )
+
+
+@TestMLRunSystem.skip_test_if_env_not_configured
+@pytest.mark.enterprise
+class TestAllKindOfServing(TestMLRunSystem):
+    project_name = "test-mm-serving-v33"
+    # Set image to "<repo>/mlrun:<tag>" for local testing
+    image: typing.Optional[str] = "docker.io/davesh0812/mlrun:1.7.0"
```
Comment (assaf758) [inline tests/system/model_monitoring/test_app.py]: Please remove

File: mlrun/model_monitoring/controller.py [hunk 3]
```diff
@@ -445,13 +445,6 @@ def model_endpoint_process(
             m_fs = fstore.get_feature_set(
                 endpoint[mm_constants.EventFieldType.FEATURE_SET_URI]
             )
-            labels = endpoint[mm_constants.EventFieldType.LABEL_NAMES]
```
Comment (assaf758) [inline mlrun/model_monitoring/controller.py:448]: why is this removed?

**Review by assaf758 — CHANGES_REQUESTED** [review]
Some minor comment inline.
Still need to check if it helps for BC to save empty list/dict

File: mlrun/model_monitoring/controller.py (hunk 3 above)
Comment (davesh0812) [reply mlrun/model_monitoring/controller.py:448]: @assaf758  currently we are not using it, and if we wish to update the fs schema we need to it from the stream pod

**Review by assaf758 — APPROVED** [review]

**Pull request merged**